content/en/docs/11/_index.en.md

```diff
@@ -174,7 +178,9 @@ ansible node1 -i inventory/hosts -b -m service -a "name=httpd state=stopped"
 
 {{% details title="Solution Task 5" %}}
 ```bash
-$ cat webhook_rulebook.yml 
+cat webhook_rulebook.yml 
+```
+```bash
```

Comment:
Maybe using the yaml syntax highlighting in all yaml code blocks could improve readability?
```suggestion
```yaml
```